### DISCUSSION TITLE
Internationalization / Localization support

### DISCUSSION THREAD

**Discussion by dylanjha** [muxinc/media-chrome#397] [Proposals]
It looks like some groundwork was laid to extract some of the hard-coded strings: https://github.com/muxinc/media-chrome/blob/main/src/js/labels/labels.js

How much work is involved to support i18n / l10n? Imagining a scenario where someone can query the media-controller element and pass in their own translations, for example:

```js
document.querySelector('media-controller').setLabels({
  VOLUME: () => 'volumen',
  SEEK_FORWARD_N_SECS: ({ seekOffset }) => `advanza ${seekOffset} segundos`
  /* ... etc */
})
```

**Comment by medienbaecker** [comment]
This would be so, so important for actual accessibility on non-english sites. Not only do you expect users to understand a second language, screen readers often pronounce the words according to the page language. This makes it impossible to understand, even to users who understand English.
